app/controllers/merchants_controller.rb

```diff
@@ -1,5 +1,5 @@
 class MerchantsController < ApplicationController
   def show
-    @merchant = Merchant.find(params[:id])
```

Comment:
Wondering why this was changed from :id to :merchant_id?

Reply:
Readability, clarification, and following the paths listed in the user story.